calibration/PandoraPFACalibrate_MipResponse.cc

```diff
@@ -226,13 +214,31 @@ void GeVToMIP::MakeHistograms()
                 TH1F *i_hMuonDirectionCorrectedCaloHitEnergy = (TH1F*) i_file->Get("MuonDirectionCorrectedCaloHitEnergy");
 
                 if (i_hECalDirectionCorrectedCaloHitEnergy!=NULL)
-                    m_hECalDirectionCorrectedCaloHitEnergy->Add(i_hECalDirectionCorrectedCaloHitEnergy,1.0);
+                    if (m_hECalDirectionCorrectedCaloHitEnergy!=NULL)
```

Comment:
Can you please format this in Pandora style of bracketing the if expression if it's followed by an else or else if.  Can you also do this for two following if statements too.